dts/arm/nordic/nrf51822.dtsi

```diff
@@ -42,6 +42,14 @@
 			reg = <0x40000000 0x1000>;
 			interrupts = <0 NRF_DEFAULT_IRQ_PRIORITY>;
 			status = "okay";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			gpregret1: gpregret1@4000051c {
```

Comment:
Missing entries in validate_base_addresses.c for these devices

Reply:
Added